core/extract/extract_parser.py

```diff
@@ -2,7 +2,7 @@
 import logging
 import re
 import dateutil.parser
-from core.util.audit_logs_parsing import Log, ConnectionLog
+from core.replay.connections_parser import ConnectionLog,Log
 from core.util.log_validation import is_valid_log, is_duplicate
 
```

Comment:
Fix for using connectionLog from `connection_parser.py` and removed `audit_logs_parsing`